### ISSUE TITLE
missing_argument_linter() incorrect for multiple missing arguments

### ISSUE THREAD

**Issue by MichaelChirico** [r-lib/lintr#2399]
```r
lint(text = "foo(,,)", linters = missing_argument_linter())
# <text>:1:5: style: [missing_argument_linter] Missing argument 1 in function call.
# foo(,,)
#     ^
# <text>:1:6: style: [missing_argument_linter] Missing argument 2 in function call.
# foo(,,)
#      ^
```

But there are three missing arguments.

**Comment by AshesITR** [comment]
Wasn't the choice deliberate to allow trailing missing argument? e.g. `rlang::list2(a, b,)`

**Comment by MichaelChirico** [comment]
No, `allow_trailing = FALSE` by default.
